roles/network_plugin/cilium/templates/cilium/cr.yml.j2

```diff
@@ -125,7 +134,22 @@ rules:
   - cilium.io
   resources:
   - ciliumcidrgroups
+  - ciliuml2announcementpolicies
+  - ciliumpodippools
+  - ciliuml2announcementpolicies/status
   verbs:
   - list
   - watch
+{% if cilium_version %} 
```

Comment:
We also need to add this for the cilium-operator clusterrole. see https://github.com/cilium/cilium/blob/dd47e9c1df59b5fd1b61d45d9a071f4175651a29/install/kubernetes/cilium/templates/cilium-operator/clusterrole.yaml#L229

Reply:
it was already there https://github.com/deveshk0/kubespray/blob/master/roles/network_plugin/cilium/templates/cilium-operator/cr.yml.j2#L125

Do i need to change anything?

Reply:
woo, sorry, I miss it. you don't need to change anything.